src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -277,7 +277,7 @@ internal static void PropagateUncaughtException (IntPtr env, IntPtr javaThreadPt
 
 			// Disabled until Linker error surfaced in https://github.com/xamarin/xamarin-android/pull/4302#issuecomment-596400025 is resolved
 			//System.Diagnostics.Debugger.Mono_UnhandledException (javaException);
-			mono_unhandled_exception (javaException);
+			//mono_unhandled_exception (javaException);
```

Comment:
In the final PR, you can remove all usage of this `mono_unhandled_exception` field.